ndbench-dyno-plugins/src/main/java/com/netflix/ndbench/plugin/dyno/DynoJedisGetSetPipeline.java

```diff
@@ -111,4 +111,32 @@ public String writeSingle(String key) throws Exception {
         return jedisUtils.pipelineWrite(key, dataGenerator, MAX_PIPE_KEYS, MIN_PIPE_KEYS);
     }
 
+    /**
+     * Perform a bulk read operation
+     * @return a list of response codes
+     * @throws Exception
+     */
+    public List<String> readBulk(final List<String> keys) throws Exception {
```

Comment:
Why do we care to implement bulk operations in the current plugins?

Reply:
just retrofitting to existing plugins. Alternatively we can throw a UnsupportedOperationException.

Reply:
Lets throw `UnsupportedOperationException` so that plugin owners can implement based on what they are looking for

Reply:
#resolved